src/linear_algebra/basic.lean

```diff
@@ -1128,6 +1126,20 @@ variables [ring α] [add_comm_group β] [add_comm_group γ] [add_comm_group δ]
 variables [module α β] [module α γ] [module α δ]
 include α
 
+@[extensionality] lemma ext {f g : β ≃ₗ[α] γ} (h : f.to_fun = g.to_fun) :
+  f = g :=
```

Comment:
I prefer chaining two exts, rather than an ext and a congr_arg. The `ext` tactic does the chaining for me...
I will try proving this with `equiv.ext` tomorrow.

Reply:
Hmmm... I tried proving this with `equiv.ext`, but I didn't get very far.

Reply:
I shortened it myself.

Reply:
Thanks!
I also tried to write it like `\forall x, f x = g x` but for some reason the coercion to function didn't trigger. This happens to me a lot... I don't know why.